frontend/src/components/DefaultInput/input.tsx

```diff
@@ -0,0 +1,16 @@
+import { InputHTMLAttributes } from "react";
+import * as S from "./styles";
+
+type InputProps = {
+} & InputHTMLAttributes<HTMLInputElement>
+
+
+export const Input = ({ name = '', type = 'text', ...rest}: InputProps) => {
```

Comment:
Nesse caso, pode só generalizar tudo como `props` e depois passar tudo ao componente.

```suggestion
export const Input = (props: InputProps) => {
```